integration_tests/features/Propagation.feature

```diff
@@ -34,7 +34,7 @@ Feature: Block Propagation
   Scenario: Duplicate block is rejected
     Given I have 1 seed nodes
     And I have a base node MINER connected to all seed nodes
-    When I mine but do not submit a block BLOCKA on MINER
+    When I mine but don't submit a block BLOCKA on MINER
```

Comment:
Should we not leave out the single quotation as an editor or IDE would view it as a the start/end of a string?